heli/assetlist.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "helichain",
+  "assets": [
+    {
+      "description": "The native token of HeliChain",
+      "denom_units": [
+        {
+          "denom": "uheli",
+          "exponent": 0
+        },
+        {
+          "denom": "heli",
+          "exponent": 6
+        }
+      ],
+      "base": "uheli",
+      "name": "Heli",
+      "display": "heli",
+      "symbol": "HELI",
+      "logo_URIs": {
```

Comment:
Please omit the `logo_URIs` property. (But keep `images`)

Reply:
I removed logo_URIs in assetlist.json